### ISSUE TITLE
Linux version requires FUSE (documentation/installer bug)

### ISSUE THREAD

**Issue by MarkBergman-cbica** [CBICA/CaPTk#257]
**Describe the bug**

The Linux version packaged with AppImage requires FUSE:

```
./captk
dlopen(): error loading libfuse.so.2

AppImages require FUSE to run. 

```

The requirement itself is fine, but this needs to be documented and the installer should check for FUSE and alert the user, rather than claiming that the install was successful.

**Comment by probonopd** [comment]
Please see https://github.com/AppImage/AppImageKit/wiki/FUSE.

**Comment by surrsurus** [comment]
Was not aware of the FUSE requirement, I made all of the packages on Ubuntu so it was always present. Will document this in the wiki and add a check to the installer.

**Comment by surrsurus** [comment]
I have added relevant documentation to the wiki and a notice in the installer if FUSE can't be found, though it still allows for the AppImage to be installed since the contents can be extracted.

**Comment by surrsurus** [comment]
Fixed in #263
